lbz/exceptions.py

```diff
@@ -2,32 +2,15 @@
 # coding=utf-8
 from http import HTTPStatus
 
-from lbz.response import Response
-
 
 class LambdaFWException(Exception):
     message = HTTPStatus.INTERNAL_SERVER_ERROR.description
     status_code = HTTPStatus.INTERNAL_SERVER_ERROR.value
-    headers = {}
-    resp = None
 
-    def __init__(self, message=""):
+    def __init__(self, message: str = ""):
         if message:
             self.message = message
 
-    def get_resp(self):
```

Comment:
I'd actually leave this method, it is heavily related to this class and it might come in handy in some weird situation. Maybe rename it to `get_lbz_response` and remove `self.resp` (so just `return Response(...)`)